doc/source/index.rst

```diff
@@ -124,22 +124,23 @@ Ray provides Python, Java, and *EXPERIMENTAL* C++ API. And Ray uses Tasks (funct
     .. code-block:: shell
 
       bazel build //cpp:ray_cpp_pkg
-      bazel build //cpp/example:example
+      cd cpp/example/
+      bazel build //:example
 
     | Option 1: run the example directly with a dynamic library path. It will start a Ray cluster automatically.
 
     .. code-block:: shell
 
       ray stop
-      ./bazel-bin/cpp/example/example --ray-dynamic-library-path=bazel-bin/cpp/example/example.so
+      ./bazel-bin/example --ray-dynamic-library-path=bazel-bin/example.so
 
     | Option 2: connect to an existing Ray cluster with a known redis address (e.g. `127.0.0.1:6379`).
 
     .. code-block:: shell
 
       ray stop
-      ray start --head --port 6379 --redis-password 5241590000000000 --node-manager-port 62665
-      ./bazel-bin/cpp/example/example --ray-dynamic-library-path=bazel-bin/cpp/example/example.so --ray-redis-address=127.0.0.1:6379
+      ray start --head --port 6379 --redis-password 5241590000000000 --node-manager-port 62665 --include-dashboard false
```

Comment:
Why do we need to add `--include-dashboard false` here?

Reply:
Because when you build ray by `pip install -e python --verbose`, dashboard will not included by default. You will see a n exception.

Reply:
But this doesn't happen for users with `pip install ray`, right?

Reply:
Right. But this tutorial in current document only supports compiling from source code now. We didn't involve this example in any release wheel yet. Maybe I can modify this tutorial after any C++ worker version releasing?

Reply:
Oh, I see.